app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -77,6 +77,8 @@ $toolbar-button-size: 50px;
 
   .govuk-textarea {
     overflow: scroll;
+    padding: 30px;
```

Comment:
We can use a mixin to produce this value using the govuk-frontend spacing scale as follows:
`padding: govuk-spacing(6);`